eng/SourceBuild.props

```diff
@@ -11,6 +11,7 @@
       <InnerBuildArgs>$(InnerBuildArgs) /p:IncludeNuGetPackageArchive=false</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) /p:IncludeAdditionalSharedFrameworks=false</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) /p:IncludeSharedFrameworksForBackwardsCompatibilityTests=false</InnerBuildArgs>
+      <InnerBuildArgs Condition="'$(SourceBuildUseMonoRuntime)' == 'true'">$(InnerBuildArgs) /p:DISABLE_CROSSGEN=true</InnerBuildArgs>
```

Comment:
When I change files for `installer`, I'm not sure if I should put them in the PR as patches, or as changes to the `installer` repo.

I manually applied this change to the generated tarball to verify the PR works.

Reply:
> When I change files for `installer`, I'm not sure if I should put them in the PR as patches, or as changes to the `installer` repo.

They should be made against the files in the installer repo.   We should never carry installer patches.  We are working hard to eliminate all patches going forward.  .NET 7.0 looks like it will only ship with three.  That all said, this type of configuration setting feels appropriate as you have implemented.